src/components/Cardsfolder/index.jsx

```diff
@@ -0,0 +1,90 @@
+import { useFetch } from "../../utils/hooks/useFetch";
+import { formatDate, generateTimeText } from "../../utils/hooks/date";
+import thumbnail from "../../assets/thumbnail.svg";
+import styles from "./index.module.css";
+import starticon from "../../assets/staricon.svg";
+import moreoptionicon from "../../assets/moreoptionicon.svg";
+import { useState } from "react";
+
+function Cardsfolder(props) {
+  const CardData = useFetch(props.url);
+  const [isHovering, setIsHovering] = useState(false);
+
+  const handleMouseOver = () => {
+    setIsHovering(true);
+  };
+
+  const handleMouseOut = () => {
+    setIsHovering(false);
+  };
+
+  return (
+    <div className={styles.card_grid_container}>
+      {CardData.data[0] ? (
+        CardData.data.map((link) => (
+          <a href={link.url}>
+            <div key={link.id} className={styles.card}>
+              <div className={styles.card_img_div} background-image>
+                {link.imageSource ? (
+                  <img
+                    src={link.imageSource}
+                    className={`${styles.card_img} ${
+                      isHovering ? styles.zoomIn : ""
+                    }`}
+                    onMouseOver={handleMouseOver}
+                    onMouseOut={handleMouseOut}
+                    alt={link.title}
+                  />
+                ) : (
+                  <img
+                    src={thumbnail}
+                    className={`${styles.card_img} ${
+                      isHovering ? styles.zoomIn : ""
+                    }`}
+                    onMouseOver={handleMouseOver}
+                    onMouseOut={handleMouseOut}
+                    alt="thumbnail_img"
+                  />
+                )}
```

Comment:
중복되는 코드가 많아서 공통화해주면 좋을 거 같아요!
```suggestion
                  <img
                    src={link.imageSource || thumbnail}
                    className={`${styles.card_img} ${
                      isHovering ? styles.zoomIn : ""
                    }`}
                    onMouseOver={handleMouseOver}
                    onMouseOut={handleMouseOut}
                    alt={link.imageSource ? link.title : "thumbnail_img"}
                  />
```